Compilation error when using older mpb

```
2020-10-07T03:37:22.7928060Z In file included from mpb.cpp:27:0:
2020-10-07T03:37:22.7929831Z adjust_verbosity.hpp: In constructor 'meep::adjust_mpb_verbosity::adjust_mpb_verbosity()':
2020-10-07T03:37:22.7930752Z adjust_verbosity.hpp:31:19: error: 'mpb_verbosity' was not declared in this scope
2020-10-07T03:37:22.7931243Z        old_level = mpb_verbosity;
2020-10-07T03:37:22.7931583Z                    ^~~~~~~~~~~~~
2020-10-07T03:37:22.7956821Z adjust_verbosity.hpp:31:19: note: suggested alternative: 'verbosity'
2020-10-07T03:37:22.7957606Z        old_level = mpb_verbosity;
2020-10-07T03:37:22.7957990Z                    ^~~~~~~~~~~~~
2020-10-07T03:37:22.7958319Z                    verbosity
```

--- a/src/adjust_verbosity.hpp
+++ b/src/adjust_verbosity.hpp
@@ -27,20 +27,19 @@ namespace meep {
 class adjust_mpb_verbosity {
 public:
   adjust_mpb_verbosity() {
-    #ifdef HAVE_MPB
+    #if defined(HAVE_MPB) && (MPB_VERSION_MAJOR > 1 || (MPB_VERSION_MAJOR == 1 && MPB_VERSION_MINOR >= 11))
       old_level = mpb_verbosity;
       mpb_verbosity = verbosity - 1;
       if (mpb_verbosity < 0) mpb_verbosity = 0;
       if (mpb_verbosity > 3) mpb_verbosity = 3;
     #else
       // avoid warnings
-      (void)amount;
       (void)old_level;
     #endif
   }
 
   ~adjust_mpb_verbosity() {
-    #ifdef HAVE_MPB
+    #if defined(HAVE_MPB) && (MPB_VERSION_MAJOR > 1 || (MPB_VERSION_MAJOR == 1 && MPB_VERSION_MINOR >= 11))
       mpb_verbosity = old_level;
     #endif
   }

